src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -191,6 +191,15 @@
   <data name="AssemblyInitializeShouldBeValidTitle" xml:space="preserve">
     <value>AssemblyInitialize methods should have valid layout</value>
   </data>
+  <data name="AssertionArgsShouldAvoidConditionalAccessDescription" xml:space="preserve">
+    <value>Assert methods should not contain conditional access.</value>
```

Comment:
Please either provide more explanation as to why this change or remove the description

Reply:
Added

Reply:
removed,